events/scheme_events/wc_plague_scheme/wc_plague_saves/wc_plague_save_events.txt

```diff
@@ -0,0 +1,2610 @@
+		triggered_desc = {
+			trigger = {
+				OR = {
+					is_close_or_extended_family_of = scope:lover
+					is_consort_of = scope:lover
+					is_employer_of = scope:lover
+					has_any_scripted_relation = scope:lover
+				}
+			}
+			desc = plague_save.0001.rel_to_me
+		}
+		desc = plague_save.0001.end
+	}
+	
+	theme = plague_scheme
+	left_portrait = {
+		character = scope:target
+		animation = grief
+	}
+	right_portrait = {
+		character = scope:lover
+		animation = pain
+	}
+	widget = {
+		gui = "event_window_widget_scheme"
+		container = "custom_widgets_container"
+	}
+	
+	trigger = {
+		scope:plague_method = flag:hired_assassin
+		scope:target = {
+			any_relation = {
+				type = lover
+				plague_save_0001_lover_trigger = yes
+			}
+		}
+	}
+
+	weight_multiplier = {
+		base = 1
+		modifier = {
+			add = 1
+			scope:target = {
+				any_relation = {
+					type = lover
+					plague_save_0001_lover_trigger = yes
+					OR = {
+						has_opinion_modifier = {
+							target = root
+							modifier = love_opinion
+						}
+						has_relation_soulmate = root
+					}
+				}
+			}
+		}
+		# Mortal Adoration Perk:
+		mortal_adoration_murder_save_modifier = yes
+	}
+
+	immediate = {
+		save_scope_value_as = {
+			name = plague_saved
+			value = flag:lover_stopped_plague
+		}
+		scope:target = {
+			random_relation = {
+				type = lover
+				limit = {
+					plague_save_0001_lover_trigger = yes
+				}
+				weight = {
+					base = 1
+					modifier = {
+						add = 5
+						OR = {
+							has_opinion_modifier = {
+								target = root
+								modifier = love_opinion
+							}
+							has_relation_soulmate = root
+						}
+					}
+				}
+				save_scope_as = lover
+			}
+		}
+		#random_dummy_gender_plague_effect
+		random_dummy_gender_assassin_effect = yes
+		plague_failure_effect = yes # Fires the response for the target
+
+	}
+
+	option = {
+		name = plague_save.0001.a
+		show_as_tooltip = {
+			if = {
+				limit = { exists = scope:scheme_discovered }
+				add_dread = medium_dread_gain
+			}
+			scope:scheme = {
+				end_scheme = yes
+			}
+		}
+		ai_chance = {
+			base = 100
+		}
+	}
+
+	option = {
+		name = plague_save.0001.b
+		show_as_tooltip = {
+			if = {
+				limit = { exists = scope:scheme_discovered }
+				add_dread = medium_dread_gain
+			}
+			start_scheme = {
+				type = murder
+				target = scope:target
+			}
+		}
+		hidden_effect = {
+			save_scope_value_as = {
+				name = restart_scheme
+				value = yes
+			}
+		}
+		ai_chance = {
+			base = 0
+		}
+	}
+
+	after = {
+		show_as_tooltip = {
+			if = {
+				limit = {
+					exists = scope:scheme_discovered
+				}
+				known_plague_sacrifice_effect = {
+					SACRIFICE = scope:lover
+					PLAGUER = root
+					EXPOSER = scope:target
+				}
+			}
+			else = {
+				unknown_plague_sacrifice_effect = {
+					SACRIFICE = scope:lover
+					PLAGUER = root
+				}
+			}
+		}
+		hidden_effect = {
+			trigger_event = plague_outcome.9999
+		}
+	}
+}
+
+
+# #############################
```

Comment:
Again clearing the branch. Commented out vanilla events.

Reply:
Done, also deleted now, didn't realize those were there